pkg/tuned/controller.go

```diff
@@ -1175,11 +1501,12 @@ func (c *Controller) changeWatcher() (err error) {
 	defer wFs.Close()
 
 	// Register fsnotify watchers.
-	for _, element := range []string{tunedBootcmdlineFile} {
+	for _, element := range []string{tunedBootcmdlineFile, tunedDeferredUpdateStatusTrigger} {
 		err = wFs.Add(element)
 		if err != nil {
 			return fmt.Errorf("failed to start watching %q: %v", element, err)
 		}
+		klog.Infof("watch enabled: %q", element)
```

Comment:
Nit: Can we be a little more clearer and say something like: "monitoring filesystem events on %q" ?  My error message above is also not ideal...

Reply:
improved both